packages/cli/test/fixtures/unit/commands/build/lambda-with-128-memory/.vercel/project.json

```diff
@@ -2,6 +2,7 @@
   "orgId": ".",
   "projectId": ".",
   "settings": {
+    "nodeVersion": "16.x",
```

Comment:
I would prefer this go in package.json engines

I'm also curious why this test needs to be pinned?

Reply:
I see now, the assertion was checking for `nodejs16.x` already. I think that can be changed to a wildcard with object.match() and that way it doesnt need to be pinned (since this test is about checking memory and not runtime)

Reply:
Removed the runtime check entirely from this test.